tests/mtouch/RegistrarTest.cs

```diff
@@ -346,6 +346,8 @@ public void MT4134 ()
 					new { Framework = "BackgroundTasks", Version = "13.0" },
 					new { Framework = "QuickLookThumbnailing", Version = "13.0" },
 					new { Framework = "AutomaticAssessmentConfiguration", Version = "13.4" },
+					// waiting on apple to ifx the headers https://github.com/xamarin/maccore/issues/2258
+					// new { Framework = "ClockKit", Version = "14.0" },
```

Comment:
I would not add the lines for this, since the fix will likely mean we don't need them (and they won't merge cleanly with other new frameworks to be added)